directory_sso_api_client/user.py

```diff
@@ -16,6 +16,7 @@ class UserAPIClient(AbstractAPIClient):
         'verify_verification': 'api/v1/verification-code/verify/',
         'user_create': 'api/v1/user/',
         'user_create_profile': 'api/v1/user/profile/',
+        'user_update_profile': 'api/v1/user/profile/update/',
```

Comment:
if you're feeling adventurous: it would be good to refactor how urls are defined. see: https://github.com/uktrade/directory-api-client/blob/master/directory_api_client/company.py#L24

if you don't want to do it in this RP then no worries :)

Reply:
I'm going to create a PR to refactor FAB so can do clean up all this together if that's ok :-)